lanterndb/lantern.ipynb

```diff
@@ -161,16 +161,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "add_embedding_job_query = \"\"\"\n",
-    "\tSET lantern_extras.openai_token = 'API_TOKEN';\n",
+    "cur.execute(\"\"\"\n",
+    "    ALTER SYSTEM SET lantern_extras.enable_daemon=true;\n",
+    "\tSELECT pg_reload_conf();\n",
+    "\"\"\")\n",
+    "cur.execute(\"\"\"\n",
+    "\tALTER DATABASE postgres SET lantern_extras.openai_token='OPENAI_KEY';\n",
+    "\"\"\")\n",
+    "cur.execute(\"\"\"\n",
     "\tSELECT add_embedding_job(\n",
     "\t    'compliance_documents',         -- Name of the table\n",
     "\t    'chunk',                        -- Source column for embeddings\n",
     "\t    'vector',                       -- Destination column for embeddings\n",
-    "\t    'openai/text-embedding-3-large' -- Embedding model to use\n",
+    "\t    'openai/text-embedding-3-small' -- Embedding model to use\n",
```

Comment:
any reason we'd prefer smaller dimensions for this blog?

Reply:
text-embedding-3-small works better with Postgres and we haven't seen notably different performance for the additional cost